src/Administration/Resources/app/administration/src/app/init-pre/store.init.ts

```diff
@@ -35,5 +35,9 @@ export default function initStore() {
      */
     if (app) {
         app.use(Store.instance._rootState);
+
```

Comment:
here is an example of how we could load pinia stores asynchronously as well, so not to have all stores registered, that would require to use `getAsync` instead of `get`, and it would load and prepare store whenever needed. Especially useful with stores which are not global